build/packages.json

```diff
@@ -26,6 +16,16 @@
         "features/polyPinion",
         "features/polyPreview",
         "features/test",
-        "podjs"
+        "platform/feature-api/api/fetch-spec",
+        "platform/feature-api/api/manifest-parser",
+        "platform/feature-api/api/pod-api",
+        "platform/feature-api/api/rdf",
+        "platform/feature-api/api/rdf-convert",
+        "platform/feature-api/api/rdf-spec",
+        "platform/feature-api/communication/bubblewrap",
+        "platform/feature-api/communication/port-authority",
+        "platform/feature-api/communication/postoffice",
+        "platform/feature-api/communication/remote-pod",
+        "platform/podjs"
```

Comment:
OK, on the face of it, it makes sense, because what's being done is to move all upstream podjs deps to where they make sense. However, the problem is that the features do have podjs as a dependency. OTOH, podjs is not tested, so errors could show up where they should not... 

There's an additional thing here, which is not new, but was there before. `demo` copies `dist/pod.js`, so that needs to be installed first. I have no idea why it didn't fail before, maybe because it's installed if it's not found. But baseline is: before, and now, we should probably move `demo` right after podjs.

The rest... I don't know. Maybe the fact that we're first doing npm ci, then build, then test, is possibly hiding some dependency graph problems, and certainly doing build and tests in that order will make errors introduced in podjs (which does not have tests) show up where they should not. 

So could you consider the possibility of putting podjs along with everything else *before* the features?

Reply:
This is **not** the order in which packages are being built, it's just all packages that are supposed to be build in alphabetical order. The actual build order is determined from parsing all the `package.json` files. _demo_ declares _podjs_ as a dependency, that's why it always worked.